### ISSUE TITLE
How to install opengl from scratch

### ISSUE THREAD

**Issue by anantha-vidhya** [Williano/OpenGL#1]
How to build opengl library to desktop from source code step by step.
Thanks & Regards 
A.N.V.Lavanya

**Comment by Williano** [comment]
Please is it a question?

**Comment by anantha-vidhya** [comment]
Yes, I am trying to  build opengl for compiling navit application which is relying on opengl. 
That is the reason i am putting this question. Once i know the steps i apply same steps to cross-compile for arm platform by exporting toolchain for arm board.
Thanks & Regards
A.N.V.Lavanya

**Comment by Williano** [comment]
https://learnopengl.com/Getting-started/Creating-a-window

This will help you
